content/en/docs/contribute/localization.md

```diff
@@ -205,14 +205,21 @@ To ensure accuracy in grammar and meaning, members of your localization team sho
 
 ### Source files
 
-Localizations must be based on the English files from the most recent release, {{< latest-version >}}.
+Localizations must be based on the English files from a specific release targeted by the localization team.
+Each localization team can decide which release to target which is referred to as the _target version_ below.
 
-To find source files for the most recent release:
+To find source files for your target version:
 
 1. Navigate to the Kubernetes website repository at https://github.com/kubernetes/website.
-2. Select the `release-1.X` branch for the most recent version.
+2. Select branch for your target version from followings.
 
-The latest version is {{< latest-version >}}, so the most recent release branch is [`{{< release-branch >}}`](https://github.com/kubernetes/website/tree/{{< release-branch >}}).
+Target version | Branch
+-----|-----
+Next version | [`dev-{{< skew nextMinorVersion >}}`](https://github.com/kubernetes/website/tree/dev-{{< skew nextMinorVersion >}})
+Latest version | [`master`](https://github.com/kubernetes/website/tree/master)
+Previous version | `release-*.**`
+
+The `master` branch holds content for the current release `{{< latest-version >}}`. The release team will create `{{< release-branch >}}` branch shortly before the next release: v{{< skew nextMinorVersion >}}. If new release branch `{{< release-branch >}}` is created before merging your localization branch to the `master` branch, switch  upstream branch for your localization branch to `{{< release-branch >}}`.
```

Comment:
I think `switch upstream branch for your localization branch to release-1.20` is not a complete guide for the case of `if new release branch release-1.20 is created before merging your localization branch to the master branch`.

I think we need to merge the localization branch into both `master` branch and new release branch to make both branches include localized contents of the localization branch.

Since [Line:279](https://github.com/kubernetes/website/pull/25795/files#diff-6f6c393e27c210c7c735bfaf19496920276feb95fc15a977ae359fb25e76f77aR276) also provides a similar note, how about remove the sentence from this section.

```suggestion
The `master` branch holds content for the current release `{{< latest-version >}}`. The release team will create `{{< release-branch >}}` branch shortly before the next release: v{{< skew nextMinorVersion >}}.
```

Reply:
👍 Thanks!